src/browser/api/application.js

```diff
@@ -524,6 +535,21 @@ Application.run = function(identity, configUrl = '' /*, callback , errorCallback
             type: 'connected',
             uuid
         });
+
+        const parentConfigUrl = coreState.getConfigUrlByUuid(uuid);
+
+        rvmBus.registerLicenseInfo({
+            licenseKey: mainWindowOpts.licenseKey,
```

Comment:
`mainWindowOpts` here is directly the `_options` object here (bad naming I guess, its what was there...) so this should be the full option set